bundle-preview/bundle-ai-chat.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div id="ai-chat-container"></div>
+    <script type="module">
+      import * as aiChat from "./static/bundles/aiChat.es.js"
+
+      const requestOpts = {
+        apiUrl: "https://api.rc.learn.mit.edu/ai/http/syllabus_agent/",
+        transformBody: (messages) => {
+          const body = {
+            collection_name: "content_files",
+            message: messages[messages.length - 1].content,
+            course_id: "course-v1:xPRO+PCDEx",
+          }
+          return body
+        },
+      }
+
+      aiChat.init(
```

Comment:
`aiChat.init` just takes all the props from AiChat component https://mitodl.github.io/smoot-design/?path=/docs/smoot-design-ai-aichat--docs